hugo/content/citation/_index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: How to cite the Genome Portal and the data
+---
+
+In line with the principles of _FAIR_ and _Open Science_, we encourage the reuse and recognition of material made available on the Swedish Reference Genome Portal.
+On this page, you will find information about how to cite the portal when reusing and referencing the content. Please note that the information on the portal is
+updated continuously, therefore it is important to refer to specific versions (or to provide access dates) within citations.
+
+Below are instructions for how to cite different aspects of the Genome Portal. These include:
+
+- Citing the the Swedish Reference Genome Portal website
+
+- Citing a specific species entry in the Genome Portal
+
+- Citing the data in the Genome Portal
+
+- Citing the underlying code of the Genome Portal, including its dependency on JBrowse 2
+
+### 1. Citing the Swedish Reference Genome Portal website
+
+The Resource Identification Portal was created in support of the <a target="_blank" href="https://www.rrids.org/">Resource Identification Initiative</a>.
+It aims to promote the identification, discovery, and reuse of research resources. Research Resource Identifiers (**RRIDs**) are persistent and unique identifiers for
+referencing a research resource. For official guidance see the [SciCrunch page on RRID citations](https://scicrunch.org/resources/about/guidelines).
+
+The RRID for the Swedish Reference Genome Portal is [**XXXXXXXX**](https://XXXXXXXXXX).
```

Comment:
I would maybe put the general information about PRIDs (that I do find very valuable) as a hover-popup or footnote, referred to by the first PRID reference on this line.